template/subspace/ansible.cfg.erb

```diff
@@ -13,4 +13,3 @@ strategy = mitogen_linear
 
 [ssh_connection]
 pipelining = True
-control_path = /tmp/subspace-control-%%h-%%p-%%r
```

Comment:
Oh sorry I should have posted it here instead of slack, but tailscale did not like me reusing ssh connections. They would randomly fail but it pretty much always failed before doing a full provision.

Reply:
Interesting.  I think this line was kindof fake news anyways I'm not sure it actually sped anything up.